packages/arb-token-bridge-ui/src/components/common/NewDialog.tsx

```diff
@@ -0,0 +1,129 @@
+import { useCallback, useRef, useState } from 'react'
+
+import { ButtonProps } from './Button'
+import { TokenApprovalDialog } from '../TransferPanel/TokenApprovalDialog'
+import { useIsOftV2Transfer } from '../TransferPanel/hooks/useIsOftV2Transfer'
+import { useSelectedToken } from '../../hooks/useSelectedToken'
+import { useIsCctpTransfer } from '../TransferPanel/hooks/useIsCctpTransfer'
+import { WithdrawalConfirmationDialog } from '../TransferPanel/WithdrawalConfirmationDialog'
+import { useArbQueryParams } from '../../hooks/useArbQueryParams'
+/**
+ * Returns a promise which resolves to an array [boolean, unknown] value,
```

Comment:
There's overlap with the current content from Dialog.tsx but NewDialog would eventually replace it